connectivity/drivers/mbedtls/FEATURE_CRYPTOCELL310/binaries/CMakeLists.txt

```diff
@@ -1,18 +1,15 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-function(_mbed_get_libcc_310_core)
-    if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
-        set(LIBCC_310_CORE TOOLCHAIN_GCC_ARM/libcc_310_core.a)
-    elseif(${MBED_TOOLCHAIN} STREQUAL "ARM")
-        set(LIBCC_310_CORE TOOLCHAIN_ARM/libcc_310_core.ar)
-    elseif(${MBED_TOOLCHAIN} STREQUAL "IAR")
-        set(LIBCC_310_CORE TOOLCHAIN_IAR/lib_cc310_ext.a)
-    endif()
-    target_link_libraries(mbed-mbedtls-cryptocell310
-        INTERFACE
-            ${CMAKE_CURRENT_SOURCE_DIR}/${LIBCC_310_CORE}
-    )
-endfunction()
+if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    set(LIBCC_310_CORE TOOLCHAIN_GCC_ARM/libcc_310_core.a)
+elseif(${MBED_TOOLCHAIN} STREQUAL "ARM")
+    set(LIBCC_310_CORE TOOLCHAIN_ARM/libcc_310_core.ar)
+elseif(${MBED_TOOLCHAIN} STREQUAL "IAR")
+    set(LIBCC_310_CORE TOOLCHAIN_IAR/lib_cc310_ext.a)
+endif()
 
-_mbed_get_libcc_310_core()
+target_link_libraries(mbed-mbedtls-cryptocell310
+    INTERFACE
+        ${CMAKE_CURRENT_SOURCE_DIR}/${LIBCC_310_CORE}
```

Comment:
```suggestion
        ${LIBCC_310_CORE}
```

Reply:
this is one of the exceptions, other other one is linker script as well. 

It's recommended to use for libraries the full path, this is quoted from CMake docs (https://cmake.org/cmake/help/latest/command/target_link_directories.html)

> Note This command is rarely necessary and should be avoided where there are other choices. Prefer to pass full absolute paths to libraries where possible, since this ensures the correct library will always be linked. The find_library() command provides the full path, which can generally be used directly in calls to target_link_libraries(). Situations where a library search path may be needed include: